### PR TITLE
Added new song

### PR DESCRIPTION
**Pull request by yaziddio**
I added a new song called Rewrite the Stars

### CODE REVIEW

File: Monika After Story/game/script-songs.rpy [hunk 1]
```diff
@@ -595,3 +595,31 @@ label mas_song_stand_by_me_long:
...
+    m 3hubfb "{i}~Cause you are the one i was meant to find~{/i}"
+    m 1ekbfa "{i}~It's up to you~{/i}"
+    m 3ekbfb "{i}~And it's up to me~{/i}"
+    m 1dkbfb "{i}~No one could say what we get to be~{/i}"
+    m 3ekbfb "{i}~So why don't we rewrite the stars~{/i}"
+    m 2ekbfa "{i}~Maybe the world could be ours~{/i}"
+    m 2dkbfa "{i}~Tonight~{/i}"
+    m 5ekbfa "The world feels like ours every time I'm with you [player]~"
+    m 5hubfa "I love you so much!"
+    return "love"
```
Comment (multimokia) [inline Monika After Story/game/script-songs.rpy]: Needs to return derandom like other short type songs
```suggestion
    return "derandom|love"
```

File: Monika After Story/game/script-songs.rpy [hunk 2]
```diff
@@ -595,3 +595,31 @@ label mas_song_stand_by_me_long:
...
+    m 3ekbfa "{i}~Say you were made to be mine~{/i}"
+    m 2fkbsa "{i}~Nothing could keep us apart~{/i}"
+    m 3hubfb "{i}~Cause you are the one i was meant to find~{/i}"
+    m 1ekbfa "{i}~It's up to you~{/i}"
+    m 3ekbfb "{i}~And it's up to me~{/i}"
+    m 1dkbfb "{i}~No one could say what we get to be~{/i}"
+    m 3ekbfb "{i}~So why don't we rewrite the stars~{/i}"
+    m 2ekbfa "{i}~Maybe the world could be ours~{/i}"
+    m 2dkbfa "{i}~Tonight~{/i}"
+    m 5ekbfa "The world feels like ours every time I'm with you [player]~"
```
Comment (multimokia) [inline Monika After Story/game/script-songs.rpy]: Need to dissolve into/out of pose 5
```suggestion
    show monika 5ekbfa at t11 zorder MAS_MONIKA_Z with dissolve
    m 5ekbfa "The world feels like ours every time I'm with you [player]~"
```

File: Monika After Story/game/script-songs.rpy [hunk 3]
```diff
@@ -595,3 +595,31 @@ label mas_song_stand_by_me_long:
...
+            aff_range=(mas_aff.NORMAL,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_rewrite_the_stars:
+    m 3ekbfb "{i}~What if we rewrite the stars~{/i}"
+    m 3ekbfa "{i}~Say you were made to be mine~{/i}"
+    m 2fkbsa "{i}~Nothing could keep us apart~{/i}"
+    m 3hubfb "{i}~Cause you are the one i was meant to find~{/i}"
```
Comment (yaziddio) [inline Monika After Story/game/script-songs.rpy]:
```suggestion
    m 3hubfb "{i}~You'd be the one i was meant to find~{/i}"
```

Comment (yaziddio) [inline Monika After Story/game/script-songs.rpy]:
```suggestion
    m 3hubfb "{i}~You'd be the one i was meant to find~{/i}"
```

Comment (multimokia) [reply Monika After Story/game/script-songs.rpy]: You don't need to review. Just push to your branch
image
image

Comment (yaziddio) [reply Monika After Story/game/script-songs.rpy]: Ok, sorry, still kinda dont understand how github works

**Comment by jmwall24** [comment]
@yaziddio please don't add stuff that is completely separate to existing PRs, and really don't add the same thing to 2 PRs.

**Pull request merged**